docs/customizing.md

```diff
@@ -69,6 +69,25 @@ of parsed commits.
 To add a new changelog type, create a new class that implements the
 [`ChangelogNotes` interface](/src/changelog-notes.ts).
 
+### Customizing Changelog Types and Releaseable Units
+
+To change the "releasable units" for your code, you can [customise the changelog sections](https://github.com/google-github-actions/release-please-action#overriding-the-changelog-sections) for your project.
```

Comment:
Perhaps something along the lines of: "A release pull request is created when a commits lands that would create an entry in the changelog. You can customize which types of commits appear in the changelog by updating the `changelog-sections` configuration."